src/r2mm/mods/ProfileImportExport.ts

```diff
@@ -49,8 +52,33 @@ async function saveDownloadedProfile(profileData: string): Promise<string> {
  * @returns string The path to the created file
  */
 async function downloadProfileCode(profileCode: string): Promise<string> {
-    const response = await ProfileApiClient.getProfile(profileCode);
-    return saveDownloadedProfile(response.data);
+    try {
+        let is404: boolean = false;
+        const response: AxiosResponse<string> = await retry(
+            () => ProfileApiClient.getProfile(profileCode),
+            5,
+            1000,
+            () => { return !is404 },
+            (e: any) => {
```

Comment:
The signature for this argument is `(e: Error|unknown)`, not `(e: any)`. Using `any` here masks hides a bug: `e.message` might be undefined, so calling `e.message.startsWith()` causes another error.

I think best way in this situation would be to do something like:
```
(e: unknown) => {
    console.error(e);
    if (R2Error.fromThrownValue(e).message.startsWith("404")) {
        is404 = true;
    }
}
```